cli/debug.go

```diff
@@ -56,6 +57,8 @@ func printKey(kv engine.MVCCKeyValue) (bool, error) {
 }
 
 func runDebugKeys(cmd *cobra.Command, args []string) error {
+	initCacheSize()
```

Comment:
Why does `runDebugKeys` do this but the other commands don't? It seems like it would be better for `openStore` to ensure that the flags have been set.